src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -39,18 +40,11 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.AsyncComplet
 {
     internal sealed class CompletionSource : ForegroundThreadAffinitizedObject, IAsyncExpandingCompletionSource
     {
-        internal const string RoslynItem = nameof(RoslynItem);
-        internal const string TriggerLocation = nameof(TriggerLocation);
-        internal const string ExpandedItemTriggerLocation = nameof(ExpandedItemTriggerLocation);
-        internal const string CompletionListSpan = nameof(CompletionListSpan);
-        internal const string InsertionText = nameof(InsertionText);
-        internal const string HasSuggestionItemOptions = nameof(HasSuggestionItemOptions);
-        internal const string Description = nameof(Description);
         internal const string PotentialCommitCharacters = nameof(PotentialCommitCharacters);
-        internal const string ExcludedCommitCharacters = nameof(ExcludedCommitCharacters);
         internal const string NonBlockingCompletion = nameof(NonBlockingCompletion);
-        internal const string TargetTypeFilterExperimentEnabled = nameof(TargetTypeFilterExperimentEnabled);
-        internal const string ExpandedItemsTask = nameof(ExpandedItemsTask);
+
+        // Don't change this property! Editor code currently has a dependency on it.
+        internal const string ExcludedCommitCharacters = nameof(ExcludedCommitCharacters);
```

Comment:
OK, turns out editor depend on this property so we have to keep it for now. Only took me one full day of debugging to figure it out...
https://devdiv.visualstudio.com/DevDiv/_git/VS-Platform?path=/src/Editor/Language/Impl/Language/AsyncCompletion/AsyncCompletionSession.cs&version=GBmain&line=1168&lineEnd=1169&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
@olegtk This feels so wrong. Any idea how can we remove this dependency?